library/src/helpers/__tests__/schema.test.ts

```diff
@@ -27,4 +120,264 @@ describe('SchemaHelpers', () => {
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['>= 2 properties']);
+    });
+
+    test('should handle max properties', () => {
+      const schema = new Schema({ maxProperties: 2 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['<= 2 properties']);
+    });
+
+    test('should handle not empty string', () => {
+      const schema = new Schema({ minLength: 1 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['non-empty']);
+    });
+
+    test('should handle not empty array', () => {
+      const schema = new Schema({ minItems: 1 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['non-empty']);
+    });
+
+    test('should handle not empty object', () => {
+      const schema = new Schema({ minProperties: 1 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['non-empty']);
+    });
+  });
+
+  describe('.isExpandable', () => {
+    test('object should be expandable', () => {
+      const schema = new Schema({ type: 'object' });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('array should be expandable', () => {
+      const schema = new Schema({ type: 'array' });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('oneOf should be expandable', () => {
+      const schema = new Schema({ oneOf: [] });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('anyOf should be expandable', () => {
+      const schema = new Schema({ anyOf: [] });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('allOf should be expandable', () => {
+      const schema = new Schema({ allOf: [] });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('not should be expandable', () => {
+      const schema = new Schema({ not: {} });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('if should be expandable', () => {
+      const schema = new Schema({ if: {} });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('then should be expandable', () => {
+      const schema = new Schema({ then: {} });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+
+    test('else should be expandable', () => {
+      const schema = new Schema({ else: {} });
+      const result = SchemaHelpers.isExpandable(schema);
+      expect(result).toEqual(true);
+    });
+  });
+
+  describe('.getCustomExtensions', () => {
```

Comment:
I would probably add a test displaying that it does not return non-extensions, i.e. if `properties` are defined. Not sure if it helps the coverage since the other test does something similar.

Reply:
Added :)